lib/chef/knife/acl_base.rb

```diff
@@ -92,8 +92,10 @@ def add_to_acl!(member_type, member_name, object_type, object_name, perms)
 
         case member_type
         when "client", "user"
-          next if ace['actors'].include?(member_name)
-          ace['actors'] << member_name
+          key = "#{member_type}s"
```

Comment:
A comment somewhere around here explaining the two API request types we can get back might be good.